subprojects/build-init/src/main/java/org/gradle/api/tasks/wrapper/Wrapper.java

```diff
@@ -114,13 +113,15 @@ public enum PathBase {
     private final DistributionLocator locator = new DistributionLocator();
     private final boolean isOffline;
     private boolean distributionUrlConfigured = false;
+    private final Property<Boolean> validateDistributionUrl = getProject().getObjects().property(Boolean.class);
 
     public Wrapper() {
         scriptFile = "gradlew";
         jarFile = "gradle/wrapper/gradle-wrapper.jar";
         distributionPath = DEFAULT_DISTRIBUTION_PARENT_NAME;
         archivePath = DEFAULT_DISTRIBUTION_PARENT_NAME;
         isOffline = getProject().getGradle().getStartParameter().isOffline();
+        validateDistributionUrl.set(true);
```

Comment:
In this particular case, it doesn't noticeably change the behavior, but we should use `.convention` in our plugins.  The difference between `convention` and `set` is that set expresses an opinion (usually by the user) and convention is the value when there's an absence of an opinion.